components/loading-spinner/README.md

```diff
@@ -1,18 +1,22 @@
-# Loading Spinner
+# Loading Spinner [d2l-loading-spinner]
 
-## d2l-loading-spinner
+The loading spinner can be used to indicate to a user that the current page/component is in a loading state, potentially when waiting for asynchronous data or during rendering.
 
+<!-- docs: start hidden content -->
 ![Loading Spinner](./screenshots/loading-spinner.gif?raw=true)
+<!-- docs: end hidden content -->
 
+<!-- docs: demo live name:d2l-loading-spinner -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/loading-spinner/loading-spinner.js';
 </script>
 
 <d2l-loading-spinner></d2l-loading-spinner>
 ```
+<!-- docs: start hidden content -->
 
-**Properties:**
+### Properties:
```

Comment:
Hmm so our properties table defaults to properties being an h3, but in this case it should actually be an h2 since `Loading Spinner` is an h1. Can either restructure the document for something like

```
h1: Loading Spinner
blurb about loading spinner
h2: Loading Spinner [d2l-loading-spinner]
other technical stuff
h3 Properties
```
or we could add another property on the comment to be something like `<!-- docs: demo live tableHeadingLevel:2 ... `.

Reply:
I discussed this with Jeff this morning and looks like we are going to try and keep the format similar for single/multi variant pages (pending some more discussion). I'll switch it to use that formatting for now!